File: examples/cpp/example1.cpp
#include <iostream>

#include "LuPNT/LuPNT.hpp"

int main(int argc, char const *argv[]) {
  std::string my_name = "Jeremy";
  std::cout << "My Name is " << my_name << std::endl;

  // Respond
  std::cout << LuPNT::Hello(my_name) << std::endl;
}

File: examples/python/example1.py
from pylupnt import hello, VectorDouble, multiply_by_scalar


def main():
    print(hello("Jeremy"))

    vd = VectorDouble([0.0, 1.0, 2.0, 3.0])
    vd2 = multiply_by_scalar(vd, 2.0)
    print(vd)
    print("Multiplied by 2.0 is:")
    print(vd2)


if __name__ == "__main__":
    main()
